### DISCUSSION TITLE
DC Voltage Amplifiers

### DISCUSSION THREAD

**Discussion by wipfli** [iontraps/awesome-iontraps#16] [Q&A] [answered]
What are commercial options for amplifying the output voltages of a [fastino](https://github.com/sinara-hw/Fastino) by a factor of 2 or so? @sergachev does [Stahl Electronics](https://www.stahl-electronics.com/) sell something like this?

**Comment by sergachev** [comment, marked as answer]
Stahl - probably no.

Commercial and low noise that I know (not verified very thoroughly):
https://www.piezodrive.com/td250-six-channel-250v-amplifier-for-piezo-tubes/
https://www.physikinstrumente.com/en/products/controllers-and-drivers/piezo-drivers-for-open-loop-operation-of-piezo-actuators/e-831-piezo-amplifier-module-601950/

Maybe one of [Sinara manufacturers](https://github.com/sinara-hw/meta/wiki/Purchasing) produce this one: https://github.com/sinara-hw/HV_AMP_8CH
